documentation/contribute/add_a_method.qmd

```diff
@@ -226,31 +205,32 @@ cat src/label_projection/methods/my_method_r/script.R
 
 :::
 
-### Required sections
+The required sections are explained here in more detail:
 
-#### Imports and libraries
+### 1. Imports and libraries
```

Comment:
There is already a step 1, step 2, step 3, .... Would it make sense to use a. b. c. here?